src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -393,6 +399,8 @@ class BaseOptionsSelector extends Component {
         const defaultConfirmButtonText = _.isUndefined(this.props.confirmButtonText) ? this.props.translate('common.confirm') : this.props.confirmButtonText;
         const shouldShowDefaultConfirmButton = !this.props.footerContent && defaultConfirmButtonText;
         const safeAreaPaddingBottomStyle = shouldShowFooter ? undefined : this.props.safeAreaPaddingBottomStyle;
+        const listContainerStyles = this.props.listContainerStyles || [{flex: 1}];
```

Comment:
```suggestion
        const listContainerStyles = this.props.listContainerStyles || [styles.flex1];
```

Reply:
nice catch, thanks!

Reply:
actually should be this.props.themeStyles.flex1 ;)

Reply:
Ah, yeah